templates/components/event_hero.html

```diff
@@ -0,0 +1,28 @@
+{% load wagtailimages_tags %}
```

Comment:
@ahosgood is components the correct home for this? Or includes? I am not clear on what the difference is.

Reply:
This is probably better in the `includes` directory as it isn't an isolated component.

Saying that, breadcrumbs IS a component so should probably be in `components` rather than `includes`...

Reply:
I'll move it over - and I can update the breadcrumbs home too.

Reply:
I've done that now. Note that I also removed some breadcrumb code from the password protected page - it was not in a block so would never have been displayed.